DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/TrackTuneParams.h

```diff
@@ -32,13 +33,34 @@ struct TrackTuneParams : public o2::conf::ConfigurableParamHelper<TrackTuneParam
   };
   AddCovType tpcCovInnerType = AddCovType::Disable;
   AddCovType tpcCovOuterType = AddCovType::Disable;
-  bool sourceLevelTPC = true;   // if TPC corrections are allowed, apply them TPC source output level (tracking/reader), otherwise in the global tracking consumers BEFORE update by external detector
+  bool sourceLevelTPC = true;    // if TPC corrections are allowed, apply them TPC source output level (tracking), otherwise in the global tracking consumers BEFORE update by external detector
+  bool applyWhenReading = false; // if true, then apply at reading tracks from the file. This better NOT used as there is no way to apply lumi-dependent scaling in the reader
   bool useTPCInnerCorr = false; // request to correct TPC inner param
   bool useTPCOuterCorr = false; // request to correct TPC outer param
   float tpcParInner[5] = {};    // ad hoc correction to be added to TPC param at the inner XRef
   float tpcParOuter[5] = {};    // ad hoc correction to be added to TPC param at the outer XRef
   float tpcCovInner[5] = {}; // ad hoc errors to be added to TPC cov.matrix at the inner XRef
   float tpcCovOuter[5] = {}; // ad hoc errors to be added to TPC outer param at the outer XRef
+  float tpcCovInnerSlope[5] = {}; // slope with respect to lumi, total correction = tpcParInner * lumi*tpcCovInnerSlope
+  float tpcCovOuterSlope[5] = {}; // slope with respect to lumi, total correction = tpcParOuter * lumi*tpcCovOuterSlope
+
+  auto getCovInnerTotal(float scale) const
+  {
+    std::array<float, 5> cov{};
+    for (int i = 0; i < 5; i++) {
+      cov[i] = tpcCovInner[i] + scale * tpcCovInnerSlope[i];
```

Comment:
I had in mind that part 
Linear in error;
cov[i] = (err[i] + scale * errSlope[i])*(err[i] + scale * errSlope[i]);
Not sure what is the most effective way  to do it - and keep the name of variables ...

Reply:
@miranov25 the meaning of the `tpcCovInner` and `tpcCovOuter` was changed to error instead of the error^2. Now the full error is defined as `[tpcCovInner + int_rate*tpcCovInnerSlope]^2`